ddtrace/internal/runtime/constants.py

```diff
@@ -32,6 +32,8 @@
 ENV = 'env'
 LANG_INTERPRETER = 'lang_interpreter'
 LANG_VERSION = 'lang_version'
+LANG = 'lang'
```

Comment:
Being that the metric names have `python` in them, what is a good motivation for adding a `lang` tag?

Reply:
Not sure. Would it help search or something like that?

Reply:
`lang` with `tracer_version` since `dd-trace-js` and `dd-trace-py` both have an `0.15.0`

Reply:
oh, sorry, read the comment wrong.... it depends, if we ever submit metrics without `python` in the metric name

Reply:
All runtime metrics a prefixed with `runtime.python` for now. Maybe this changes in the future. My take is that `lang` would be redundant here and so we should remove it.

Reply:
#1068 :)

Reply:
uses the same dogstatsd client, which sets these constant tags.